docs/Troubleshooting/MySqlSizeIssue.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "MySQL Troubleshooting"
+sidebar_position: 4
+description: >
+  MySQL Troubleshooting
+---
+
+### How to delete old MySQL binlog files?
```

Comment:
It would be better to describe the symptom before we address the cause:
"Why the size of the database grows drastically?" sth like that

Reply:
But actually here is talking about how to resolve